configure.ac

```diff
@@ -471,78 +472,12 @@ if test x$with_homespool != x; then
 		MAILPATH should be set to the filename of the spool mailbox
 		relative the the home directory.
 		use: configure --with-homespool=FILE])
-	AC_DEFINE(USE_DOTLOCK,1,[ Define to use dotlocking for mailboxes. ])
-	mutt_cv_setgid=no
 else
 	AC_ARG_WITH(mailpath, AS_HELP_STRING([--with-mailpath=DIR],[Directory where spool mailboxes are located]),
-		[mutt_cv_mailpath=$withval],
```

Comment:
@gahr aah, i see. But still, maybe it would be tidier if it is in a separate commit? (the po changes)